### ISSUE TITLE
//fixlighting doesnt work properly under water

### ISSUE THREAD

**Issue by lusu007** [IntellectualSites/FastAsyncWorldEdit#615]
# Bug report for FastAsyncWorldEdit 1.16.3

**[REQUIRED] FastAsyncWorldEdit Configuration Files:** 

https://athion.net/ISPaster/paste/view/ac830d20c6884b2fbc3819c515d2ae47

**[REQUIRED] FastAsyncWorldEdit Version Number:** 

```
[15:43:35 INFO]: FastAsyncWorldEdit-357 created by Empire92, MattBDev, IronApollo, dordsor21 and NotMyFault
[15:43:35 INFO]: FAWE version:
- Date Sep 14, 2020 12:00:00 AM
- Commit f2936578
- Build 357
- Platform Bukkit
[15:43:35 INFO]: Wiki: https://wiki.intellectualsites.com/FastAsyncWorldEdit/index
[15:43:35 INFO]: ------------------ Platforms ------------------
* Bukkit-Official(1.16-357;d6c9a88)
[15:43:35 INFO]: ---------------- Capabilities ----------------
GAME_HOOKS: Bukkit-Official
CONFIGURATION: Bukkit-Official
USER_COMMANDS: Bukkit-Official
PERMISSIONS: Bukkit-Official
WORLDEDIT_CUI: Bukkit-Official
WORLD_EDITING: Bukkit-Official
```

**[REQUIRED] Spigot/Paper Version Number:** 

git-Paper-196

**[REQUIRED] Description of the problem:**

If you want to use //fixlighting to fix some underwater lighting issues //fixlighting just illuminates the whole place. 

The two screenshots are before and after //fixlighting. In this case //fixlighting should change nothing because lighting is already correct.

![Before //fixlighting](https://user-images.githubusercontent.com/10341994/93094085-bc725b00-f6a1-11ea-9402-ae711f946ff9.png)
![After //fixlighting](https://user-images.githubusercontent.com/10341994/93094093-bf6d4b80-f6a1-11ea-9387-1e108eed73d3.png)

**How to replicate**:

Just fly to a vanilla lit ocean, lake or swamp and execute //fixlighting.

**Plugins being used on the server:**

Arceon, BuildersUtilities, BungeeTabListPlus, Essentials, FastAsyncWorldEdit, goBrush, goPaint, HeadDatabase*, Hyperverse, LegendEssentials, LuckPerms, Vault, VoidGenerator, VoxelSniper, WorldEdit, WorldGuard

**Checklist**:

- [x] I included all information required in the sections above
- [x] I made sure there are no duplicates of this report [(Use Search)](https://github.com/IntellectualSites/FastAsyncWorldEdit/issues?q=is%3Aissue)
- [x] I made sure I am using an up-to-date version of [FastAsyncWorldEdit for 1.16.2](https://ci.athion.net/job/FastAsyncWorldEdit-1.16/)
- [x] I made sure the bug/error is not caused by any other plugin

**Comment by MattBDev** [comment]
So the command made the entire ocean light up?

**Comment by lusu007** [comment]
Yes exactly

**Comment by dordsor21** [comment]
Fixed in 3d2052b
